src/cata_imgui.cpp

```diff
@@ -1055,3 +1055,49 @@ void cataimgui::EndRightAlign()
 {
     ImGui::EndTable();
 }
+
+void cataimgui::init_colors()
+{
+    ImGuiStyle &style = ImGui::GetStyle();
+
+    style.Colors[ImGuiCol_Text]                   = c_white;
```

Comment:
Can we put it to some kind of config, so it won't be hardcoded?

Reply:
The colours being loaded are from the player's existing theme config.

I'm not sure that we want to expose all the specifics of the ImGui theme to the player as it may change across ImGui versions.

Reply:
We are not using just any unspecified ImGui version - we are using specific in-repo version of it. Anyway, if ImGui version will have some breaking changes you would need change either the code here or config. Storing things in configuration would spare us from recompiling this file.

Reply:
I think that eventually we will want to give the user more direct control over the ImGui theme itself. We should also think about having a semantic theme instead of merely a color theme. Instead of having some random elements of the UI be red and letting the user customize what “red” means, let the user choose what color a broken limb is independently of the colors used to compare item stats. And whatever set of colors we let the user change we should have an in–game UI for customizing them. It can save to the same JSON file, but the user should be able to see their changes immediately instead of making them restart the game.

But none of that is worth delaying this patch. This patch looks great to me.